src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/integration/RecallsIntegrationTest.kt

```diff
@@ -132,7 +133,7 @@ class RecallsIntegrationTest : IntegrationTestBase() {
       .jsonPath("$.recallLength").isEqualTo("FOURTEEN_DAYS")
       .jsonPath("$.agreeWithRecallRecommendation").isEqualTo(agree)
       .jsonPath("$.lastReleasePrison").isEqualTo(prison)
-      .jsonPath("$.recallEmailReceivedDateTime").isNotEmpty
+      .jsonPath("$.recallEmailReceivedDateTime").value(endsWith("Z"))
```

Comment:
Would be nice if we could assert on actual values here (using Fixed clock or joda time), and possibly see the output of the response json so we can see when/if the format changes.  But possibly one for after our testing discussion next week